R/render-plot.R

```diff
@@ -187,7 +192,7 @@ renderPlot <- function(expr, width = 'auto', height = 'auto', res = 72, ...,
     outputFunc,
     renderFunc,
     outputArgs,
-    cacheHint = list(userExpr = get_expr(q), res = res)
+    cacheHint = list(userExpr = installedFuncExpr(func), res = res)
```

Comment:
We should have a test for the correct `userExpr` for `renderPlot()` -- if in the future we make changes that result in the wrong cache hint, it could cause problems for users.